src/fmu/ensemble/ensemble.py

```diff
@@ -1427,11 +1433,12 @@ def get_smry(
                 not enough memory to keep all summary files in memory.
             start_date: str or date with first date to include.
                 Dates prior to this date will be dropped, supplied
-                start_date will always be included.
+                start_date will always be included. Overriden if time_index
+                is 'first' or 'last'.
             end_date: str or date with last date to be included.
                 Dates past this date will be dropped, supplied
                 end_date will always be included. Overriden if time_index
-                is 'last'.
+                is 'first' or 'last'.
             include_restart: boolean sent to libecl for wheter restarts
```

Comment:
whether while we are at it...

Reply:
you mean e.g. `Overridden whether time_index is 'first' or 'last'`? Isn't the use of whether over if best if these two options are the only options?

Reply:
No, I mean 'wheter' is an independent typo :), should be 'whether'.